libraries/apollo-tooling/src/test/kotlin/com/apollographql/apollo3/tooling/InspectionQueryTest.kt

```diff
@@ -100,16 +99,115 @@ class IntrospectionQueryTest {
       }
     }""".trimIndent()
 
+  private val normalizedV3IntrospectionQuery = """
+    query IntrospectionQuery {
+      __schema {
+        queryType {
+          name
+        }
+        mutationType {
+          name
+        }
+        subscriptionType {
+          name
+        }
+        types {
+          ...FullType
+        }
+        directives {
+          name
+          description
+          locations
+          args {
+            ...InputValue
+          }
+        }
+      }
+    }
+
+    fragment FullType on __Type {
+      kind
+      name
+      description
+      fields(includeDeprecated: true) {
+        name
+        description
+        args {
+          ...InputValue
+        }
+        type {
+          ...TypeRef
+        }
+        isDeprecated
+        deprecationReason
+      }
+      inputFields {
+        ...InputValue
+      }
+      interfaces {
+        ...TypeRef
+      }
+      enumValues(includeDeprecated: true) {
+        name
+        description
+        isDeprecated
+        deprecationReason
+      }
+      possibleTypes {
+        ...TypeRef
+      }
+    }
+
+    fragment InputValue on __InputValue {
+      name
+      description
+      type {
+        ...TypeRef
+      }
+      defaultValue
+    }
+
+    fragment TypeRef on __Type {
+      kind
+      name
+      ofType {
+        kind
+        name
+        ofType {
+          kind
+          name
+          ofType {
+            kind
+            name
+            ofType {
+              kind
+              name
+              ofType {
+                kind
+                name
+                ofType {
+                  kind
+                  name
+                  ofType {
+                    kind
+                    name
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }""".trimIndent()
+
   private fun String.normalized(): String {
     return Buffer().writeUtf8(this).use {
-      it.parseAsGQLDocument(null).getOrThrow().toUtf8()
+      it.parseAsGQLDocument(null).getOrThrow().toUtf8().trimIndent()
```

Comment:
Is the trimIndent() required here?

Reply:
Ah, I see below why it's required. I think if it's called `normalize`, the expectation is to normalize both the expected and actual results. If we want to compare formating and whitespaces, we should call this function "format". But this sounds more fragile

Reply:
Yeah it didn't really make sense 😅 Just pushed comparing parsed with original json, but ignoring spaces